lib/computer_player.rb

```diff
@@ -0,0 +1,14 @@
+class ComputerPlayer
+  attr_reader :token
+
+  def initialize(token:)
+    @token = token
+  end
+
+  def selection(board)
+    loop do
+      selection = rand(1..9)
+      return selection if board.is_available?(selection)
+    end
```

Comment:
I sense some duplication between our two player objects. Perhaps there is a responsibility hiding amongst them?

Reply:
added a SelectionValidator class to validate the selection